frontends/precinct-scanner/src/screens/scan_warning_screen.test.tsx

```diff
@@ -115,10 +119,12 @@ test('undervote by 1', () => {
 
   screen.getByRole('heading', { name: 'Review Your Ballot' });
   screen.getByText(
-    new RegExp(`You can vote for more people for ${contest.title}.`)
+    new RegExp(`You may vote for more candidates for: ${contest.title}.`)
```

Comment:
The undervote screen uniquely provides an `or` option between the two buttons. Was this intentional? Let's make this consistent between other review screens.

Reply:
It was intentional when the buttons were on one line, but I see that the overvote screen didn't have this. fixed.